recipes-support/net-snmp/net-snmp_5.7.3.bb

```diff
@@ -70,6 +71,8 @@ do_install_append() {
     install -d ${D}${systemd_unitdir}/system
     install -m 0644 ${WORKDIR}/snmpd.service ${D}${systemd_unitdir}/system
     install -m 0644 ${WORKDIR}/snmptrapd.service ${D}${systemd_unitdir}/system
+    install -m 0644 ${WORKDIR}/environmentfile ${D}${sysconfdir}/default/snmpd
+    install -m 0644 ${WORKDIR}/environmentfile_trapd ${D}${sysconfdir}/default/snmptrapd
```

Comment:
oh well.. debatable. could be more user-friendly to have a file in /etc/snmp/ already